keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -379,7 +382,28 @@ command or function to work in the same way in another tool. This can
 further enhance the usability and adoption of different cluster
 manager.
 
-### Notes/Constraints/Caveats (Optional)
+### Notes/Constraints/Caveats
+
+#### How to Utilize the API
+This section provides guidance on the recommended way to interact with the API. It is suggested that
+all clusterProfile objects, which represent a single cluster in the inventory, should be emitted
+on a dedicated management/hub Kubernetes cluster. This approach allows consumers to have a single
```

Comment:
This approach allows "Multicluster controllers to have a global view of the clusters of the ClusterSet".

Reply:
I would like to not assume that the consumer is a controller, it could be some UI code, script or even human.